Reading List/Reading List/Views/BookTableViewCellDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  BookTableViewCellDelegate.swift
+//  Reading List
+//
+//  Created by Cora Jacobson on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+protocol BookTableViewCellDelegate: AnyObject {
```

Comment:
Great job figuring out that the delegate needs to be constrained to the AnyObject protocol in order to be weak (and thus avoid a retain cycle)